deployment/base/scripts/insertDefaults.php

```diff
@@ -283,5 +293,28 @@ function getVodPackagerUrl()
 	return $vodPackagerUrl;
 }
 
+function isSupportedTemplateFile($dirName)
+{
+	echo("File [$dirName] does not exist - checking if filename support template file\n");
+	
+	// escaping the dot for preg_match to avoid using it as a single wildcard character
+	$supportedTemplateFilenames = array(
+		'01\.Partner',
+		'07\.DeliveryProfile'
+	);
+	
+	$fileName = basename($dirName);
+	
+	foreach ($supportedTemplateFilenames as $supportedTemplateFilename)
+	{
+		if (preg_match("/^$supportedTemplateFilename\w*\.ini/", $fileName) === 1)
```

Comment:
Why not add \w*\.ini to the supportedTemplateFilenames I think it will be much cleaner that way

Reply:
I agree